src/Behavior/Binary.php

```diff
@@ -79,7 +79,7 @@ public function rewriteCondition(Condition $condition, $relation = null)
          * {@see \ipl\Orm\Compat\FilterProcessor::requireAndResolveFilterColumns()}
          */
         $column = $condition->metaData()->get('columnName');
-        if (isset($this->properties[$column])) {
+        if (isset($column) && isset($this->properties[$column])) {
```

Comment:
Using `isset()` on a variable that just holds a function’s return value is not idiomatic. A strict comparison with `null` is clearer and more appropriate because it expresses that you only care about distinguishing `null` from any other value, not about existence vs. non‑existence.
Furthermore, this is also a change in behavior, as `$array[null]` is treated as `$array[‘’]` in previous versions. I have no objection to this change, but it should be justified in the commit description.

This change is also not yet included in the PR description.